module/VuFindSearch/tests/unit-tests/src/VuFindTest/Backend/Primo/BackendTest.php

```diff
@@ -182,6 +182,83 @@ public function testMergedParamBag()
         $back->search(new Query('baz'), 0, 10, $myParams);
     }
 
+    /**
+     * Data provider for testPcAvailabilityFilter
+     *
+     * @return array
+     */
+    public function getPcAvailabilityData(): array
+    {
+        return [
+            [
+                '',
+                true
+            ],
+            [
+                true,
+                true
+            ],
+            [
+                1,
+                true
+            ],
+            [
+                '1',
+                true
+            ],
+            [
+                'true',
+                true
+            ],
+            [
+                false,
+                false
+            ],
+            [
+                0,
+                false
+            ],
+            [
+                '0',
+                false
+            ],
+            [
+                'false',
+                false
+            ],
+        ];
+    }
+
+    /**
+     * Test pcAvailability filter.
+     *
+     * @dataProvider getPcAvailabilityData
+     *
+     * @return void
+     */
+    public function testPcAvailabilityFilter($value, $expected): void
+    {
+        $params = new ParamBag(
+            [
+                'filterList' => [
+                    'pcAvailability' => [
+                        'values' => [
+                            $value
+                        ]
+                    ]
+                ]
+            ]
+        );
+        $expectedParams = ['limit' => 10, 'pageNumber' => 1, 'filterList' => [], 'pcAvailability' => $expected, 'query' => [['index' => null, 'lookfor' => 'foo']]];
```

Comment:
While we don't enforce long line checking in tests, It might be worth wrapping this and line 256 below for readability.

Reply:
True. I went throught the whole file and wrapped long lines while at it.

Reply:
Thanks!